spec/delayed/job_groups/job_group_spec.rb

```
@@ -225,4 +225,26 @@
       running_job.should_not have_been_destroyed
     end
   end
+
+  describe "#failure_cancels_group?" do
+    let(:failure_cancels_group) { true }
+
+    before do
+      job_group.update_attribute(:failure_cancels_group, failure_cancels_group)
```

Should do `update_attributes!` so saves won't silently fail in tests.